afkak/test/fixtures.py

```diff
@@ -147,7 +147,7 @@ def __init__(self, host, port, broker_id, zk_host,
 
         self.replicas = replicas
         self.partitions = partitions
-        self.min_insync_replicas = partitions // 2 + 1
+        self.min_insync_replicas = replicas // 2 + 1
```

Comment:
WTH?

Reply:
Oh good, that's not one of mine! :-)

Reply:
Looks like that fixed it. The failure now is the docs build.

Reply:
Yaaaah!

Reply:
So fun question: how did it ever work on 0.8.2.2?